### DISCUSSION TITLE
Problem with file upload (local)

### DISCUSSION THREAD

**Discussion by joldenburgSOLITA** [mckaywrigley/chatbot-ui#1311] [Help]
Hi, I'm running this on Windows, everything is seemingly set up ok, but whenever I try to upload a file I get this error: 

"Failed to process file".

**Comment by joldenburgSOLITA** [comment]
![image](https://github.com/mckaywrigley/chatbot-ui/assets/119294091/63c98099-2261-48fa-9a36-824c39305c18)

**Reply by Yeghro** [reply]
I'm also dealing with this issue.  Trying to figure out why communication is no good between app and database.  I can access the backend just fine and use are able to be created and logged in but working with the database seems to have issues.  Maybe a permission thing?   If any figures something out or has a solution please chime in.
